Setting/Factions/The Coven.md

```diff
@@ -0,0 +1,32 @@
+---
+tier: "5"
+allies:
+  - "[[Scrappers]]"
+enemies:
+  - "[[Ship Defense Corps]]"
+  - Intership Council
+demonym: cadmus
+aliases:
+  - CADMUS
+---
+#faction
+
+## Background
+They call themselves the Coalition for Biomorphological Equality, but to anyone else they are the Coven, a disorganized group aggressively bent on the democratization of biomodification and freedom to use morphogenic compounds. Those cogent enough often stage violent protests, of [[The Coven|CADMUS]] exclusive research. The [[Ship Defense Corps]] have been known to break up such altercations, by whatever means necessary.
+
+## Turf
+They have small encampments in various parts of the ship but the largest by far are [[The Boroughs]] A mire of fungal growths and makeshift housing, interspersed with the occasional 'clinic'. These surround [[The Academy]]
+
+## NPCs
+[["Doc" ]] - Once an esteemed surgeon, left the academy to share his knowledge with the people. Not nearly as much of a zealot, but does their best to care for those whom experience more dire consequences of experimentation.
+
+[[Ursula]] - Youthful escaped test subject whom found a home amonst those who helped her get by with the side effects of brutal mind research. Afflicted with [[Psychogenic Encephalopathy]] which gives light psionic ability but is experiencing a deteriorating psychosis.
+
+[[Bones]] A scrapper who among them knows who pays the best for biological scrap
+## Notable Assets
```

Comment:
Was struggling to think if there were particular assets they might have that would be interesting in a game context. This seemed possibly something that could be thought about in play

Reply:
Maybe a bio bomb, or a captured computer terminal? A cache of resources for the coming mutiny?